src/styles/utils/_keyframes.scss

```
@@ -0,0 +1,9 @@
+@keyframes rotate {
+  from {
+    transform: rotate(89deg);
+  }
+
+  to {
+    transform: rotate(449deg);
+  }
+}
```

Цей keyframe виглядає занадто складно і не зрозуміло
має виглядати ось так
```suggestion
@keyframes rotate {
  from {
    transform: rotate(0deg);
  }

  to {
    transform: rotate(360deg);
  }
}
```
для того щоб інші стилі працювали правильно - треба в кожної стрілки поміняти ширину і висоту місцями, задати `transform-origin: bottom` і замінити `top: 50%` на `bottom: 50%` для стрілок